addons/frag/CfgAmmo.hpp

```diff
@@ -340,21 +101,7 @@ class CfgAmmo {
         caliber = 2.8;
     };
 
-    class GVAR(spall_small): GVAR(small) {
-        timeToLive = 0.1;
-    };
-
-    class GVAR(spall_medium): GVAR(medium) {
-        timeToLive = 0.15;
-    };
-
-    class GVAR(spall_large): GVAR(large) {
-        timeToLive = 0.25;
-    };
-
-    class GVAR(spall_huge): GVAR(huge) {
-        timeToLive = 0.3;
-    };
-
-    #include "CfgAmmoReflections.hpp"
+    #include "cfgAmmoSpall.hpp"
+    #include "cfgAmmoFragSpawner.hpp"
+    #include "cfgAmmoFragParameters.hpp"
```

Comment:
Change files to start with a capital letters.

Reply:
Fixed